easy-ui-react/package.json

```diff
@@ -41,8 +41,10 @@
     "overlayscrollbars": "^2.2.1",
     "overlayscrollbars-react": "^0.5.1",
     "react-aria": "^3.26.0",
+    "react-is": "^18.2.0",
```

Comment:
this is an official package from facebook for determining react types at runtime. used in this case for reading `children`